static/style.css

```
@@ -42,7 +42,8 @@ h1 {
 }
 
 #container-graph .caption {
-  max-width: 502px;
+  width: 492px;
+  height: 72px;
```

Increase font size for better readability.
```suggestion
  width: 492px;
  height: 72px;
  font-size: 14pt;
```